Lecture5/Task1/Helper/CalcProcessing.java

```diff
@@ -0,0 +1,31 @@
+package Helper;
+
+public class CalcProcessing {
+
+    private CalcProcessing() {
+    }
+
+    public static double getResultByOperation(double firstNumber, double secondNumber, char operation) {
+        if (operation == '+') {
+            return Operations.getSum(firstNumber, secondNumber);
+        } else if (operation == '-') {
+            return Operations.getDiff(firstNumber, secondNumber);
+        } else if (operation == '*') {
+            return Operations.getDer(firstNumber, secondNumber);
+        } else {
+            return Operations.getQuo(firstNumber, secondNumber);
+        }
+    }
+
+    public static String getResultTextByOperation(char operation) {
```

Comment:
not a good design since you have to parse user input twice. Instead, encapsulate user operation in separate object, e.g. 
class SummOperation {

  public static double calculate(double op1, double op2);
}
public String toString() {}.

Reply:
fixed
ResultsFactory used